packages/ts/react-crud/test/TextFieldController.ts

```diff
@@ -19,8 +19,8 @@ export default class TextFieldController {
     child: HTMLElement,
     user: ReturnType<(typeof userEvent)['setup']>,
   ): Promise<TextFieldController> {
-    const instance = await waitFor(() => child.closest('vaadin-text-field')!);
-    return new TextFieldController(instance, user);
+    const instance = await waitFor(() => child.parentElement!);
+    return new TextFieldController(instance as TextFieldElement, user);
```

Comment:
The new test abstractions are rather difficult to deal with. For now I just made `TextFieldController` work with any field, we can consider refactoring this in a separate PR to make it more generic / flexible.